packages/history-service/src/create-root-location-transformer.ts

```diff
@@ -22,6 +27,10 @@ export interface RootLocationTransformer {
     currentRootLocation: history.Location,
     consumerUid: string
   ): history.LocationDescriptorObject;
+
+  createLocation(
```

Comment:
We should find a better name for this method. How about `createRootLocationForMultipleConsumers`?

Reply:
done